rero_ils/accounts_views.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Invenio Account custom views."""
+
+from invenio_accounts.views.rest import LoginView as CoreLoginView
+from invenio_accounts.views.rest import use_kwargs
+from invenio_userprofiles.models import UserProfile
+from sqlalchemy.orm.exc import NoResultFound
+from webargs import ValidationError, fields
+from werkzeug.local import LocalProxy
+
+
+#
+# Field validators
+#
+def user_exists(email):
+    """Validate that a user exists."""
+    try:
+        # print('======================>', email)
+        profile = UserProfile.get_by_username(email)
+        # email = profile.user.email
+    except NoResultFound:
+        if not current_datastore.get_user(email):
+            raise ValidationError('USER_DOES_NOT_EXIST')
+
+current_datastore = LocalProxy(
+    lambda: current_app.extensions['security'].datastore)
```

Comment:
As other LocalProxy of the poject, these lines could be moved at the top of the file, just after imports